lib/susedistribution.pm

```diff
@@ -202,7 +202,10 @@ sub init_desktop_runner {
     if (!check_screen('desktop-runner', $timeout)) {
         record_info('workaround', "desktop-runner does not show up on $hotkey, retrying up to three times (see bsc#978027)");
```

Comment:
That bug ref is the one that's been linked to the failing tests; as you already stated: the softfail info does not match at all - so it should probably be improved (if even still being valid)

Reply:
ok :+1:

Reply:
Not sure if I understood the improvement right, to remove it.
I guess for KDE it's clear that KRunner is not running and for other desktop it's kind of the same.
We have to retry because of performance or system has to finish start.

Reply:
The workaround makes sense - but the recorder softfail info with the bug report does not make sense; that part should be corrected/adjusted imho

Reply:
It's just record_info.  https://openqa.opensuse.org/tests/3118260#step/opensuse_welcome/7

Reply:
yes - clear - but it records info referring to a bug number that is not accurate (as you stated yourself: """ The linked bug is pretty outdated and IMO not related to KDE""")

Reply:
True, but the retry is generic, specific to KDE is that KRunner has to start and it needs longer than other DMs.
I would leave the bug there as it's the "only" reference.

Reply:
ok, I removed it, it's mentioned in the commit and it's old/ not related